AlsongDalsong/7-6/소스.c

```diff
@@ -0,0 +1,20 @@
+﻿/*관계연산자가 비트단위연산자보다 우선순위이고 비트단위연산자의 경우 좌결합성이다!*/
+#include <stdio.h>
+
+int main(void)
+{
+	unsigned int a = 0;
+	unsigned int b = 0;
+	unsigned int c = 0;
+
+	printf("a:");
+	scanf("%u", &a);
+	printf("b:");
+	scanf("%u", &b);
+	printf("c:");
+	scanf("%u", &c);
+
+	printf("a & b == b & c의 값은 %u입니다.", a & b == b & c);
```

Comment:
연산자의 우선순위에 따라서 `==`이 `&` 보다 먼저 평가될 수도 있습니다. 아래와 같이 의미를 보다 명확하게 표현해주세요.
```suggestion
	printf("a & b == b & c의 값은 %u입니다.", (a & b) == (b & c));
```

Reply:
예제에서 왜 답이 예상과 다르게 1이 되는지는에 대한 연산자 우선순위 해석문제(1라인 주석처리)